vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1368,6 +1369,36 @@ void LedgerImpl::GetConfirmationsHistory(
       to_timestamp_seconds, callback);
 }
 
+void LedgerImpl::RemoveData(int32_t remove_mask,
+      ledger::OnDataRemovedCallback callback) {
+  ledger_client_->DeleteData(remove_mask,
+      GetReconcileStamp(),
+      std::bind(&LedgerImpl::OnRemoveData,
+                this,
+                callback,
+                _1));
+}
+
+void LedgerImpl::OnRemoveData(
+    ledger::OnDataRemovedCallback callback,
+    bool result) {
+  callback(result);
+}
+
+void LedgerImpl::GetAutoContributeCount(
+    ledger::GetAutoContributeCountCallback callback) {
+  ledger_client_->RetrieveAutoContributeCount(
+      GetReconcileStamp(),
+      std::bind(&LedgerImpl::OnGetAutoContributeCount, this, callback, _1, _2));
```

Comment:
why are you calling `OnGetAutoContributeCount` instead of just calling the callback right here?

Reply:
Because I need `count` and `previouse_reconcile_stamp` as args for `callback`